lib/README.md

```diff
@@ -1,4 +1,6 @@
-# Gutenberg PHP
+	# Gutenberg PHP
```

Comment:
Is this tab indentation intentional? I assumed not 😄

Reply:
Um, yes? :trollface: